packages/demo-dynamic/store.tsx

```diff
@@ -0,0 +1,58 @@
+import {configureStore, createAction, createSlice, ThunkAction} from '@reduxjs/toolkit';
+import {Action} from 'redux';
+import {createWrapper, HYDRATE} from 'next-redux-wrapper';
+
+const hydrate = createAction(HYDRATE);
```

Comment:
I think typing the generic will mean you won't have to cast as `any` on line 23 (otherwise it thinks hydrate is `ActionCreatorWithoutPayload` and therefore payload is undefined, I'm assuming that's why you had to cast it?). Might be useful to show in this demo?

```suggestion
const hydrate = createAction<AppState>(HYDRATE);
```

Reply:
Good suggestion, thank you! Included.